TrackierSDK/sdk-core/src/main/java/com/trackier/sdk/Util.kt

```diff
@@ -157,4 +159,23 @@ object Util {
             return null
         }
     }
+    
+    fun campaignData(context: Context, res: ResponseData) {
+        setSharedPrefString(context,Constants.SHARED_PREF_AD,res.ad)
```

Comment:
```suggestion
        setSharedPrefString(context, Constants.SHARED_PREF_AD, res.ad)
```
proper spaces between the arguments is missing

Reply:
Done